docs/network.md

```diff
@@ -0,0 +1,78 @@
+# 7. Adding new edges to routing tables
+
+This section covers the process of adding new edges, received from another nodes,
+to the routing table. It consists of several steps covered below.
+
+## 7.1 Step 1
+`PeerManagerActor` receives `RoutingTableSync` message containing list of new `edges` to add.
```

Comment:
This is the message that is generated by Step 5, right?

Reply:
@mm-near Yes. Once we merge those two PRs. I'll have to go through all points together, and do rewrite, to make sure everything fits.